packages/playwright/src/common/test.ts

```diff
@@ -290,10 +290,14 @@ export class TestCase extends Base implements reporterTypes.TestCase {
       return 'skipped';
 
     const failures = results.filter(result => result.status !== 'skipped' && result.status !== 'interrupted' && result.status !== this.expectedStatus);
+    const skipped = results.filter(result => result.status === 'skipped' && result.status !== this.expectedStatus);
+    const passed = results.filter(result => result.status === 'passed');
     if (!failures.length) // all passed
       return 'expected';
     if (failures.length === results.length) // all failed
       return 'unexpected';
+    if (failures.length && skipped.length && !passed.length) // some failed, none succeeded and the rest were skipped
```

Comment:
I believe what we need here is instead of lines 297-300 something like:

```js
if (failures.length && !passed.length)
  return 'unexpected';
```
as we don't care if there were any "skips".

Reply:
Fixed :+1: